.claude/commands/code-review-readability.md

```
@@ -121,29 +110,31 @@ Your role is to:
+   ## Warnings
+   - `file.ts:line` - [warning type] - [description]
+
+   ## Summary
+   X critical issues, Y warnings
+   ```
 
-### Summary
-X critical issues, Y warnings
-```
+2. **Return only**:
+   ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)